src/gt4sd/algorithms/generation/diffusion/core.py

```diff
@@ -318,3 +318,31 @@ def list_versions(cls) -> Set[str]:
             .list_versions()
             .union({cls.algorithm_version})
         )
+
+
+@ApplicationsRegistry.register_algorithm_application(DiffusersGenerationAlgorithm)
+class GeoDiffGenerator(DiffusersConfiguration):
```

Comment:
let's inherit `get_target_description` to describe the prompt structure and let's a accept a dictionary as prompt that we use to create `Data` object in the pipeline implementation.

Reply:
what do you mean by `describe the prompt` here?

Reply:
This method is used to describe the structure of the target parameter. In this case I would simply describe the structure of the dictionary.
You can check other configuration to get an example.

Reply:
See for example this one: https://github.com/GT4SD/gt4sd-core/blob/9fbee27141a70d5ce648f1f8ce7e10bd33900871/src/gt4sd/algorithms/controlled_sampling/advanced_manufacturing/core.py#L135